src/main.js

```diff
@@ -17,6 +17,7 @@ register("pt", () => import(/* webpackChunkName: "pt" */ "./lang/pt.json"));
 register("ru", () => import(/* webpackChunkName: "ru" */ "./lang/ru.json"));
 register("tr", () => import(/* webpackChunkName: "tr" */ "./lang/tr.json"));
 register("ja", () => import(/* webpackChunkName: "ja" */ "./lang/ja.json"));
+register("ur", () => import(/* webpackChunkName: "ja" */ "./lang/ur.json"));
```

Comment:
Can you modify `/* webpackChunkName: "ja" */` to `/* webpackChunkName: "ur" */` as well? 🙂

Reply:
Oh, sorry I missed that. Will send a fix.

Reply:
I have corrected the code. Kindly review.